java-frontend/src/main/java/org/sonar/java/se/MethodYield.java

```diff
@@ -43,6 +43,8 @@
   int resultIndex;
   @Nullable
   Constraint resultConstraint;
+  @Nullable
+  String exceptionType;
```

Comment:
As I'm not handling implicit exceptions yet, then the answer is no.
I still need a way to differentiate an exceptional path which isn't aware of the exception causing it, from a happy path.